#!/usr/bin/env python
PACKAGE = "state_estimation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("system_noise_speed",     double_t, 0, "system_noise_speed",      0, 0, 1000)
gen.add("system_noise_yaw_rate",     double_t, 0, "system_noise_yaw_rate",      0, 0, 1000)
gen.add("system_noise_acceleration",     double_t, 0, "system_noise_acceleration",      0, 0, 1000)
gen.add("measurement_noise_angular_speed_left",       double_t, 0, "measurement_noise_angular_speed_left",  1.0186, 0, 10)
gen.add("measurement_noise_angular_speed_right",       double_t, 0, "measurement_noise_angular_speed_right",  1.0186, 0, 10)
gen.add("measurement_noise_acceleration",       double_t, 0, "measurement_noise_acceleration",  0.00050125, 0, 1)
gen.add("measurement_noise_yaw_rate",       double_t, 0, "measurement_noise_yaw_rate",   0.000033018, 0, 1)

exit(gen.generate(PACKAGE, "state_estimation", "StateEstimation"))
